flake.nix

```diff
@@ -1,5 +1,5 @@
 {
-  description = "rocket-chip";
+  description = "vector";
```

Comment:
flake.nix:2 description = "vector"; should be something but vector? not sure, just a thought